### PR TITLE
Improve the quality of comments in the documents

### PR DESCRIPTION
**Pull request by yuxiaobo96**
Signed-off-by: yuxiaobo <yuxiaobogo@163.com>

### Ⅰ. Describe what this PR did
Step by step improvement comments.

### Ⅱ. Does this pull request fix one issue?

NONE

### Ⅲ. Why don't you add test cases (unit test/integration test)? (你真的觉得不需要加测试吗？)

Don't need.

### CODE REVIEW

**Comment by yunfeiyanggzq** [comment]
LGTM

**Comment by Starnop** [comment]
Thanks for your good job.  And could you please help to fix all problems of `the lack of dots for comments ` for Dragonfly in this PR?  😄

**Comment by yuxiaobo96** [comment]
Of course.

File: client/utils.go [hunk 1]
```diff
@@ -35,7 +35,7 @@ func ensureCloseReader(resp *Response) error {
 	if resp != nil && resp.Body != nil {
 		defer resp.Body.Close()
 
-		// close body ReadCloser to make Transport reuse the connection
+		// close body ReadCloser to make Transport reuse the connection.
```
Comment (yeya24) [inline client/utils.go]: If you add dot here, you need to use uppercase `Close`

File: dfdaemon/config/config.go [hunk 2]
```diff
@@ -247,29 +247,29 @@ func (u *URL) unmarshal(unmarshal func(interface{}) error) error {
...
 
-// MarshalYAML implements yaml.Marshaller to print the url
+// MarshalYAML implements yaml.Marshaller to print the url.
 func (u *URL) MarshalYAML() (interface{}, error) {
 	return u.String(), nil
 }
 
 // CertPool is a wrapper around x509.CertPool, which can be unmarshalled and
-// constructed from a list of filenames
+// constructed from a list of filenames.
```
Comment (yeya24) [inline dfdaemon/config/config.go]: Uppercase `Consgructed`

File: dfget/core/uploader/peer_server.go [hunk 3]
```diff
@@ -64,7 +64,7 @@ func newPeerServer(cfg *config.Config, port int) *peerServer {
 // ----------------------------------------------------------------------------
 // peerServer structure
 
-// peerServer offer file-block to other clients
+// peerServer offers file-block to other clients
```
Comment (yeya24) [inline dfget/core/uploader/peer_server.go]: Add dot here

File: dfget/core/uploader/uploader.go [hunk 4]
```diff
@@ -54,14 +54,14 @@ var (
 // -----------------------------------------------------------------------------
 // dfget server functions
 
-// WaitForShutdown wait for peer server shutdown
+// WaitForShutdown waits for peer server shutdown
```
Comment (yeya24) [inline dfget/core/uploader/uploader.go]: add dot

File: supernode/daemon/mgr/dfgettask/manager.go [hunk 5]
```diff
@@ -96,7 +96,7 @@ func (dtm *Manager) Add(ctx context.Context, dfgetTask *types.DfGetTask) error {
 		return err
 	}
 
-	// the default status of DfgetTask is WAITING
+	// the default status of DfgetTask is WAITING.
```
Comment (yeya24) [inline supernode/daemon/mgr/dfgettask/manager.go]: `The`

File: supernode/daemon/mgr/task/manager_test.go [hunk 6]
```diff
@@ -121,11 +121,11 @@ func (s *TaskMgrTestSuite) TestUpdateTaskInfo(c *check.C) {
 	resp, err := s.taskManager.Register(context.Background(), req)
 	c.Check(err, check.IsNil)
 
-	// return error when taskInfo equals nil
+	// return error when taskInfo equals nil.
```
Comment (yeya24) [inline supernode/daemon/mgr/task/manager_test.go]: `Return`

File: supernode/daemon/mgr/task/manager_test.go [hunk 7]
```diff
@@ -121,11 +121,11 @@ func (s *TaskMgrTestSuite) TestUpdateTaskInfo(c *check.C) {
...
 	err = s.taskManager.Update(context.Background(), resp.ID, nil)
 	c.Check(errortypes.IsEmptyValue(err), check.Equals, true)
 
-	// return error when taskInfo.CDNStatus equals ""
+	// return error when taskInfo.CDNStatus equals "".
```
Comment (yeya24) [inline supernode/daemon/mgr/task/manager_test.go]: `Return`

File: supernode/daemon/mgr/task/manager_test.go [hunk 8]
```diff
@@ -154,7 +154,7 @@ func (s *TaskMgrTestSuite) TestUpdateTaskInfo(c *check.C) {
 	c.Check(task.CdnStatus, check.Equals, types.TaskInfoCdnStatusSUCCESS)
 	c.Check(task.FileLength, check.Equals, int64(2000))
 
-	// do not update if origin CDNStatus equals success
+	// do not update if origin CDNStatus equals success.
```
Comment (yeya24) [inline supernode/daemon/mgr/task/manager_test.go]: `Do`

File: supernode/server/piece_error_bridge.go [hunk 9]
```diff
@@ -48,7 +48,7 @@ func (s *Server) handlePieceError(ctx context.Context, rw http.ResponseWriter, r
 		return errors.Wrap(errortypes.ErrEmptyValue, "dstPid")
 	}
 
-	// fulfill the taskID and pieceRange if they are empty
+	// fulfill the taskID and pieceRange if they are empty.
```
Comment (yeya24) [inline supernode/server/piece_error_bridge.go]: Fulfill

File: test/command/cmd.go [hunk 10]
```diff
@@ -140,7 +140,7 @@ func (s *Starter) getCmdGo(dir string, running time.Duration, args ...string) (c
 	return s.execCmd(running, supernodePath, args...)
 }
 
-// Supernode starts supernode.
+// Supernode starts a supernode.
```
Comment (yeya24) [inline test/command/cmd.go]: I don' t think we need `a` here.

File: test/environment/env.go [hunk 11]
```diff
@@ -30,19 +30,19 @@ var (
...
-	// TLSConfig is default tls config
+	// TLSConfig is default TLS config.
 	TLSConfig = client.TLSConfig{}
 
-	// GateWay default gateway for test
+	// GateWay is default gateway for test
 	GateWay = "192.168.1.1"
 
-	// Subnet default subnet for test
+	// Subnet is default subnet for test.
```
Comment (yeya24) [inline test/environment/env.go]: is the default

File: version/version.go [hunk 12]
```diff
@@ -64,7 +64,7 @@ var (
 	// SupernodeVersion is the version of supernode
 	SupernodeVersion = version
 
-	// DFVersion the global instance of DragonflyVersion
+	// DFVersion is the global instance of DragonflyVersion
```
Comment (yeya24) [inline version/version.go]: dot

Comment (yuxiaobo96) [reply version/version.go]: Thanks

File: dfdaemon/config/config.go (hunk 2 above)
Comment (yuxiaobo96) [reply dfdaemon/config/config.go]: This line and the previous line are one sentence. 
Don't need to update.

Comment (yeya24) [reply dfdaemon/config/config.go]: Sorry.

**Comment by yuxiaobo96** [comment]
/assign @Starnop 

I have modified all the remaining documents.

File: client/request.go [hunk 13]
```diff
@@ -41,7 +41,7 @@ func (e RespError) Error() string {
 	return e.msg
 }
 
-// Code returns the response  code
+// Code returns the response  code.
```
Comment (Starnop) [inline client/request.go]: delete the redundant space here.

File: dfdaemon/config/config.go [hunk 14]
```diff
@@ -288,22 +288,22 @@ func (cp *CertPool) unmarshal(unmarshal func(interface{}) error) error {
...
 }
 
-// MarshalYAML implements yaml.Marshaller to print the cert pool
+// MarshalYAML implements yaml.Marshaller to print the cert pool.
 func (cp *CertPool) MarshalYAML() (interface{}, error) {
 	return cp.Files, nil
 }
 
-// Regexp is simple wrapper around regexp.Regexp to make it unmarshallable from a string
+// Regexp is simple wrapper around regexp.Regexp to make it unmarshallable from a string.
```
Comment (Starnop) [inline dfdaemon/config/config.go]: s/ is simple wrapper/ is a simple wrapper

Comment (Starnop) [reply dfdaemon/config/config.go]: And add a space after the comma.

File: dfdaemon/config/config.go [hunk 15]
```diff
@@ -288,22 +288,22 @@ func (cp *CertPool) unmarshal(unmarshal func(interface{}) error) error {
...
 type Regexp struct {
 	*regexp.Regexp
 }
 
-// NewRegexp returns new Regexp instance compiled from the given string
+// NewRegexp returns new Regexp instance compiled from the given string.
```
Comment (Starnop) [inline dfdaemon/config/config.go]: s/ returns new Regexp instance/ returns a new Regexp instance

File: docs/cli_reference/dfdaemon.md [hunk 16]
```diff
@@ -30,6 +30,6 @@ dfdaemon [flags]
 
 ### SEE ALSO
 
-* [dfdaemon gen-doc](dfdaemon_gen-doc.md)	 - Generate Document for dfdaemon command line tool with MarkDown format
-* [dfdaemon version](dfdaemon_version.md)	 - Show the current version of dfdaemon
+* [dfdaemon gen-doc](dfdaemon_gen-doc.md)	 - Generate Document for dfdaemon command line tool with MarkDown format.
```
Comment (Starnop) [inline docs/cli_reference/dfdaemon.md]: In fact, the files under the `docs/cli_reference` is generated automatically by pouchrobot. 😄

File: supernode/server/result_info.go [hunk 17]
```diff
@@ -23,7 +23,7 @@ import (
 	"github.com/dragonflyoss/Dragonfly/pkg/errortypes"
 )
 
-// ResultInfo identify a struct that will returned to the client.
+// ResultInfo identify a struct that will return to the client.
```
Comment (Starnop) [inline supernode/server/result_info.go]: s/ return/ be returned

File: test/environment/env.go [hunk 18]
```diff
@@ -30,19 +30,19 @@ var (
 	// SupernodeDownloadPort is the port that supernode will listen.
 	SupernodeDownloadPort = 8009
 
-	// DragonflySupernodeBinary is default binary
+	// DragonflySupernodeBinary is default binary.
```
Comment (Starnop) [inline test/environment/env.go]: s/is default binary/ is the  default binary path

File: test/environment/env.go [hunk 19]
```diff
@@ -30,19 +30,19 @@ var (
...
 
 	// DragonflyAddress is default address dragonfly supernode listens on.
 	DragonflyAddress = fmt.Sprintf("tcp://127.0.0.1:%d", SupernodeListenPort)
 
-	// TLSConfig is default tls config
+	// TLSConfig is default TLS config.
 	TLSConfig = client.TLSConfig{}
 
-	// GateWay default gateway for test
+	// GateWay is default gateway for test
```
Comment (Starnop) [inline test/environment/env.go]: Add a dot in the end.

Comment (Starnop) [reply test/environment/env.go]: s/is default/ is the default

**Comment by yuxiaobo96** [comment]
/assign @Starnop 
Please help me.

**Comment by Starnop** [comment]
> /assign @Starnop
> Please help me.

Next time, you could request a review as follows:
+ @someone PTAL
+ @someone request a review

😄

**Comment by Starnop** [comment]
LGTM.

**Pull request merged**